ambuda/translations/sa/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,183 @@
+# Sanskrit translations for PROJECT.
```

Comment:
@suhasm and @vvasuki -- would love your help on the translations here, as I'm not very familiar with Sanskrit as used on websites.